patches/@changesets+apply-release-plan+5.0.5.patch

```diff
@@ -77,11 +67,3 @@ index 9e9d03c..fff3e27 100644
    let cwd = packages.root.dir, touchedFiles = [];
    const packagesByName = new Map(packages.packages.map((x => [ x.packageJson.name, x ])));
    let {releases: releases, changesets: changesets} = releasePlan;
-@@ -177,6 +177,7 @@ async function applyReleasePlan(releasePlan, packages, config$1 = config.default
-     type: type
-   }))), finalisedRelease = releaseWithChangelogs.map((release => versionPackage(release, versionsToUpdate, {
-     updateInternalDependencies: config$1.updateInternalDependencies,
-+    ignoreRange,
```

Comment:
missed this one?

Reply:
it seemed unused

Reply:
and was also not present in the .prod bundle build